config.go

```diff
@@ -1 +1,105 @@
 package main
+
+import (
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	flags "github.com/btcsuite/go-flags"
+	"path/filepath"
+	"io/ioutil"
+	"os"
+)
+
+const (
+	defaultConfigFilename = "lnwallet.conf"
+	defaultDataDirname    = "test_wal"
+	defaultRPCPort        = 10009
+	defaultSPVMode        = false
+	defaultPeerPort       = 10011
+	defaultBTCDHost       = "localhost:18334"
+	defaultBTCDUser       = "user"
+	defaultBTCDPass       = "passwd"
+	defaultBTCDCACertPath = ""
```

Comment:
This can be safely set with something along the lines of:
```
btcdHomeDir = btcutil.AppDir("btcd", false)
defaultBTCDCACertPath = filepath.Join(btcdHomeDir, "rpc.cert")
```